Rock/Security/Authorization.cs

```diff
@@ -166,8 +166,13 @@ public static bool Load()
         /// </summary>
         /// <param name="entityTypeId">The entity type identifier.</param>
         /// <param name="entityId">The entity identifier.</param>
-        public static void ReloadEntity( int entityTypeId, int entityId )
+        public static void ReloadEntity( int entityTypeId, int entityId, RockContext rockContext = null )
         {
+            if ( rockContext == null )
+            {
+                rockContext = new RockContext();
+            }
+
```

Comment:
This check for null and creation of context here isn't really needed since it's just being passed to `RefreshEntity()` and that will create it if it's null, and with the `using` which is even better.